.github/workflows/build_and_test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     uses: ./.github/workflows/_build.yml
     with:
       ecr-docker-image-base: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/xla_base
-      gcr-docker-image: gcr.io/tpu-pytorch/xla_base:latest
+      gcr-docker-image: gcr.io/tpu-pytorch/xla_base:cuda_12_0
```

Comment:
I wonder where you saw "The CI GPU instance expects CUDA 12.0"

Reply:
It's the new GPU autocast testing, that silently crashed the CI runner. The last trace I got indicated it, but couldn't really dig deeper. I have a feeling that some of GPU test hangs you've experienced might be also related.

Reply:
After trying to repro the error, I am realizing that the CI runner crashes and actually comes back online after waiting for a while. I think the version mismatch might have been a warning , I will continue the base image update effort in https://github.com/pytorch/xla/pull/5635, for CUDA 12 -- as our team will probably migrate sometime soon in the near future. I will land this one with the current CI image. 

cc @JackCaoG @vanbasten23